hack/release/README.md

```diff
@@ -2,17 +2,18 @@
 
 ## Steps
 
-- checkout the repository on the correct branch and changeset
-- Run localy the `make VERSION=x.v.z pre-release`
+- Checkout the repository on the correct branch and changeset `git checkout -b vX.Y.Z`
+- Run localy the command `make VERSION=x.v.z pre-release`
 - Commit all the changes generated from the previous command:
     ```console
     $ git add .
     $ git commit -s -m "release vX.Y.X"
     ```
+- Uncomment the job `push_latest_to_docker_hub` in `.gitlab-ci.yml` in case you plan to push the image to dockerhub with the `latest` image tag.
 - Add release tag, correct format: `git tag vX.Y.Z`
 - Push the generated commit and tag to the repostory branch.
     ```console
-    $ git push origin vX.Y.Z
+    $ git push origin tag vX.Y.Z
```

Comment:
I usually don't add "tag".  it was a problem for you without?

Reply:
yes I had this
```
error: src refspec v0.2.1 matches more than one.
error: failed to push some refs to 'git@github.com:DataDog/datadog-operator.git'
```